climpred/metrics.py

```diff
@@ -81,7 +162,20 @@ def _pearson_r(forecast, reference, dim=None, **metric_kwargs):
     return pearson_r(forecast, reference, dim=dim, weights=weights, skipna=skipna)
 
 
-# TODO: all metrics: docs
+_pearson_r = Metric(
```

Comment:
I overwrite the name of the function with the metric. I could also change this and then this would get the documentatation to the state of before this PR

Reply:
You could just drop the underscore here and name them `pearson_r` and so on for the actual Class instantiation of the function. Is there any reason to avoid this?

Reply:
Or could use dunderscore (double underscore) for the function and single for the class metric here.

Reply:
shouldnt the first letter of classes be capitalized? but then `get_metric_class` wouldnt work anymore. I think it would be good to not overwrite the function here. 

why did we use these underscores in the first place? Because underscore functions are functions that the user shouldnt call themselves ideally. which is the case here also.

I will change to `__pearson_r(Metric)` and `_pearson_r` as function `__pearson_r.function`.

Reply:
I think the convention is that the class itself should be CamelCase (as Metric is) but you can create objects from that class however you like. So you don't need PearsonR for instance.